cmd/optimizer-nri-plugin/main.go

```diff
@@ -237,13 +239,13 @@ func main() {
 
 			cfg = flags.Args.Config
 
-			log = logrus.StandardLogger()
-			log.SetFormatter(&logrus.TextFormatter{
-				PadLevelText: true,
-			})
+			// FIXME(thaJeztah): ucontainerd's log does not set "PadLevelText: true"
+			_ = log.SetFormat(log.TextFormat)
+			ctx := log.WithLogger(context.Background(), log.L)
```

Comment:
This currently uses a different format than what was used before; 

- Now uses `RFC3339NanoFixed` format
- Now uses `FullTimestamp: true`
- :x: does not use `PadLevelText: true`


https://github.com/containerd/log/blob/v0.1.0/context.go#L153-L158


```go
	case TextFormat:
		L.Logger.SetFormatter(&logrus.TextFormatter{
			TimestampFormat: RFC3339NanoFixed,
			FullTimestamp:   true,
		})
		return nil
```